sql/src/test/java/io/crate/operation/collect/StatsTablesTest.java

```diff
@@ -138,4 +143,35 @@ public void testUniqueOperationIdsInOperationsTable() throws Exception {
         assertTrue(queue.contains(new OperationContextLog(ctxA, null)));
 
     }
+
+    @Test
+    public void testRemoveExpiredLogs() throws Exception {
+        NodeSettingsService nodeSettingsService = new NodeSettingsService(Settings.EMPTY);
+        Settings settings = Settings.builder()
+            .put(CrateSettings.STATS_ENABLED.settingName(), true).build();
+        StatsTables stats = new StatsTables(settings, nodeSettingsService);
+
+        JobContextLog log1 = new JobContextLog(new JobContext(UUID.fromString("067e6162-3b6f-4ae2-a171-2470b63dff01"), "select 1", 1L), null);
```

Comment:
think we could also consider adding a package private VisibleForTesting constructor which takes the ended time as argument to avoid having to mock the classes